src/wasm/wasm-binary.cpp

```diff
@@ -1511,23 +1524,27 @@ void WasmBinaryBuilder::readTableElements() {
 
 void WasmBinaryBuilder::readNames() {
   if (debug) std::cerr << "== readNames" << std::endl;
+  auto name_type = getU32LEB();
+  auto subsection_size = getU32LEB();
+  WASM_UNUSED(subsection_size);
```

Comment:
absolutely! And that actually caught a silly bug in my non-binaryen code, so thanks for the suggestion!